serverless-plugin/index.js

```diff
@@ -1,37 +1,31 @@
 'use strict'
 
 const _ = require('lodash')
+const Ajv = require('ajv')
+
 const alarms = require('./alarms')
 const dashboard = require('./dashboard')
+const { pluginConfigSchema, slicWatchSchema } = require('./config-schema')
 const defaultConfig = require('./default-config')
 const CloudFormationTemplate = require('./cf-template')
 
+const ServerlessError = require('serverless/lib/serverless-error')
+const normalizeAjvErrors = require('serverless/lib/classes/ConfigSchemaHandler/normalizeAjvErrors')
```

Comment:
I am wondering how _safe_ (i.e. backward-compatible) this is...

What happens if Serverless framework shuffles things around internally in a new version and these modules are not there anymore?

We currently have `2.x` of `serverless` as a peer dependency... Should we move it to an actual regular dependency?

BTW, this is minor for now. Maybe just food for thought. We could probably hit the bullet and go ahead with this. If there's a breaking change we can deal with it later and find an appropriate solution when we have more context...

Reply:
Good point! TIL Ajv has an `errorsText` function so I've replaced it with this!